src/App.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
-import TimerArea from './component/TimerArea';
+import TimeArea from './component/TimerArea';
 
 const App = () => (
-  <div>
-    <TimerArea />
+  <div className="container">
```

Comment:
You should use the React Bootstrap Container component instead.
https://react-bootstrap.github.io/layout/grid/#container